diff --git a/components/latex_block/latex_block.jsx b/components/latex_block/latex_block.jsx
index 727ceea3a..c2d9169d7 100644
--- a/components/latex_block/latex_block.jsx
+++ b/components/latex_block/latex_block.jsx
@@ -36,7 +36,7 @@ export default class LatexBlock extends React.Component {
         }
 
         try {
-            const html = this.state.katex.renderToString(this.props.content, {throwOnError: false, displayMode: true});
+            const html = this.state.katex.renderToString(this.props.content, {throwOnError: false, displayMode: true, maxExpand: 100000});
 
             return (
                 <div
